src/test/js/shared/custom_steps.js

```diff
@@ -82,7 +82,7 @@ module.exports = function() {
         this.dontSeeInCurrentUrl('attemptfive');
         this.fillField('username', email);
         this.fillField('password', '111111111111111111');
-        this.scrollPageToBottom();
+        //XB: this.scrollPageToBottom();
```

Comment:
what does XB mean?

Reply:
I am not sure about this ,this was there in old branch where sauce labs code was written, I tried rerunning test on sauce labs by using existing code in preview branch, which is working fine , So I am removing this and replacing with code in preview branch